docs/docs/03-syntax-and-usage/09-template-composition.md

```diff
@@ -66,17 +66,17 @@ The use of the `{ children... }` expression in the child component.
 
 # Components as parameters
 
-Components can also be passed as parameters and rendered using the `{! component }` expression.
+Components can also be passed as parameters and rendered using the `@component` expression.
```

Comment:
Thinking about this, it probably needs some more description.

If `l` and `r` are template parameters, this syntax is correct, but if you have a `templ l() { <div>Hello</div> }`, then you need to use `@l()`, not just `@l`.

Reply:
I've not changed the description, but maybe these examples now make it clearer.